src/main/java/org/jabref/gui/desktop/JabRefDesktop.java

```diff
@@ -298,13 +298,15 @@ public static void openConsole(File file, PreferencesService preferencesService,
      * @return The path to the directory
      */
     public static String getDefaultFileChooserDirectory() {
-        // Property "user.home" might be "difficult" on Windows
-        // See https://stackoverflow.com/a/586917/873282 for a longer discussion
-        // The proposed solution is to use Swing's FileSystemView
-        // See https://stackoverflow.com/a/32914568/873282
-        // As of 2022, System.getProperty("user.home") returns c:\Users\USERNAME on Windows 10, whereas
-        // the FileSystemView returns C:\Users\USERNAME\Documents, which is the "better" directory
-        return FileSystemView.getFileSystemView().getDefaultDirectory().getPath();
+        if (System.getProperty("os.name").toLowerCase().contains("win")) {
+            // Property "user.home" might be "difficult" on Windows
+            // See https://stackoverflow.com/a/586917/873282 for a longer discussion
+            // As of 2022, System.getProperty("user.home") returns c:\Users\USERNAME on Windows 10
+            // but we want  C:\Users\USERNAME\Documents, which is the "better" directory
+            return System.getProperty("user.home") + "\\Documents";
```

Comment:
This is a bad idea. The original awt code calls some native OS functions for determining the correct directory. I debugged through this. And it's platform independent.

Reply:
I'm welcome for suggestions, but as you debugged through this you should know this will try to start the swing toolkit which is not a good idea and brings a lot of overhead. Since it's only the initial directory of the file selector dialog, it should be fine if it's not perfect.

Reply:
This is used as default file directory if the user starts JabRef and has not yet configured any file directory.
Ask @koppor  for the rationale behind it

Reply:
If you want to get your hands dirty. Really dirty... Use JNI: https://stackoverflow.com/a/586917 😱

Reply:
I don't want to get my hands dirty at all ;-). Let's discuss this further in https://github.com/JabRef/jabref/pull/9222.